src/rust/receiver-mock/src/metrics.rs

```diff
@@ -0,0 +1,47 @@
+use std::net::IpAddr;
+use std::sync::{Arc, Mutex};
+
+pub fn handle_carbon2(
+    lines: std::str::Lines,
+    address: IpAddr,
+    stats: &Arc<Mutex<crate::statistics::Statistics>>,
```

Comment:
I would rather use `crate::statistics::Statistics` and `Statistics`

Reply:
done